src/components/Board.vue

```diff
@@ -127,6 +127,15 @@ export default Vue.extend({
 			const { width, height } = this.ctx.canvas;
 			this.ctx.clearRect(0, 0, width, height);
 		},
+
+		saveBoard() {
+			const url = this.ctx.canvas.toDataURL('image/png').replace(/^data:image\/[^;]/, 'data:application/octet-stream');
```

Comment:
```suggestion
		saveBoard(mimeType = 'image/png') {
			const url = this.ctx.canvas.toDataURL(mimeType).replace(/^data:image\/[^;]/, 'data:application/octet-stream');
```
To allow changing to `jpeg` or something